appveyor.yml

```diff
@@ -2,9 +2,10 @@ version: 6.0.0-alpha.18-{build}
 
 image: Visual Studio 2015
 
+# cache version - netcoreapp.2.0.0-preview1-001913-00
```

Comment:
`005724` is the build of `dotnet-cli` we are currently using, but the `netcoreapp` version is actually [`2.0.0-preview1-001913-00`](https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.App/2.0.0-preview1-001913-00).